components/kyma-environment-broker/internal/provisioner/graphqlizer.go

```diff
@@ -184,8 +179,39 @@ func (g *Graphqlizer) KymaConfigToGraphQL(in gqlschema.KymaConfigInput) (string,
 		{{- end }}
 	}`)
 }
+
+func (g *Graphqlizer) marshal(obj interface{}) string {
+	var out string
+
+	val := reflect.ValueOf(obj)
+
+	switch val.Kind() {
+	case reflect.Map:
+		s, err := g.genericToGraphQL(obj, `{ {{- range $k, $v := . }}{{ $k }}:{{ marshal $v }},{{ end -}} }`)
+		if err != nil {
+			return ""
```

Comment:
if an error occurs, labels will be empty and we will have no idea why beacause lack of logs